scripts/guessingGame.js

```diff
@@ -0,0 +1,23 @@
+class guessingGame {
+  constructor(answer = Math.floor(Math.random() * 100) + 1) {
```

Comment:
since this is a relatively lengthy default value calculation, i may find a way to extract this out 👍